libgnucash/engine/ScrubBudget.h

```diff
@@ -0,0 +1,39 @@
+/********************************************************************\
+ * ScrubBudget.h -- fix budget amount signs                         *
+ *                                                                  *
+ * This program is free software; you can redistribute it and/or    *
+ * modify it under the terms of the GNU General Public License as   *
+ * published by the Free Software Foundation; either version 2 of   *
+ * the License, or (at your option) any later version.              *
+ *                                                                  *
+ * This program is distributed in the hope that it will be useful,  *
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of   *
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the    *
+ * GNU General Public License for more details.                     *
+ *                                                                  *
+ * You should have received a copy of the GNU General Public License*
+ * along with this program; if not, contact:                        *
+ *                                                                  *
+ * Free Software Foundation           Voice:  +1-617-542-5942       *
+ * 51 Franklin Street, Fifth Floor    Fax:    +1-617-542-2652       *
+ * Boston, MA  02110-1301,  USA       gnu@gnu.org                   *
+ *                                                                  *
+\********************************************************************/
+
+#include <glib.h>
+#include <qofbook.h>
+
+/* ================================================================ */
+
+/** Fix budget signs
+ * A guard is set if we have completed reversal, or there are no
```

Comment:
Say "feature".

But speaking of guards, this header needs them. You do that by adding immediately under the license block
```
#ifndef _GNC_SCRUBBUDGET_H_
#define _GNC_SCRUBBUDGET_H_
```
and then at the end
```
#endif // _GNC_SCRUBBUDGET_H_
```

The purpose is to avoid re-declaration errors if the header is included more than once in a compilation unit.